package/network/services/hostapd/files/hostapd.sh

```diff
@@ -663,7 +663,7 @@ hostapd_set_bss_options() {
 		set_default wps_independent 1
 
 		wps_state=2
-		[ -n "$wps_configured" ] && wps_state=1
+		[ -n "$wps_not_configured" ] && wps_state=1
```

Comment:
Shouldn't this be:
[ -n "$wps_not_configured" ] || wps_state=1

Configured is opposite from not_configured...

Reply:
`wps_state=2` is "enabled and configured".
`wps_state=1` is "enabled and not configured".

since `wps_configured` is never set, `wps_state` is always 2.
but when `encryption` is 'none', the script was intended to assume that wps is [not configured](https://github.com/openwrt/openwrt/pull/3781/files#diff-0524cbcb28eb9c9b7a08dd8c38f9b88ab8dab0af51bae8ca63e1124090008907L572) and set `wps_state=1`.

without this change, that feature doesn't work.